api/environments/dynamodb/dynamodb_wrapper.py

```diff
@@ -5,24 +5,34 @@
 from boto3.dynamodb.conditions import Key
 from django.conf import settings
 from django.core.exceptions import ObjectDoesNotExist
-from flag_engine.api.document_builders import build_identity_document
+from flag_engine.api.document_builders import (
+    build_environment_document,
+    build_identity_document,
+)
+
+from environments.models import Environment
 
 if typing.TYPE_CHECKING:
     from environments.identities.models import Identity
 
 
-class DynamoIdentityWrapper:
+class DynamoWrapper:
+    TABLE_NAME: str = None
```

Comment:
I don't think this should be capitalised? By convention, only constants should be capitalised like this.

Reply:
I think it's all the constants, even if they are class attributes? Although I personally don't like this either

Reply:
I'd say this isn't _really_ a constant though, right? For me, it's analogous to e.g. the `queryset` or the `permission_classes` attribute on a `ModelViewSet` class in DRF.

Reply:
Yeah, that makes sense - it felt odd